src/test/java/seedu/address/model/activity/AmountTest.java

```diff
@@ -16,9 +16,7 @@ public void constructor_invalidAmount_throwsIllegalArgumentException() {
     @Test
     public void isValidAmount() {
         // invalid amounts
-        assertFalse(Amount.isValidAmount(0)); // Not positive
         assertFalse(Amount.isValidAmount(-1)); // Negative int
-        assertFalse(Amount.isValidAmount(0.004999)); // Rounds to 0
```

Comment:
Refer to comment for `ExpenseCommandParser.java`

Reply:
Not sure what you mean.